cmd/abapEnvironmentPullGitRepo.go

```diff
@@ -178,7 +178,7 @@ func readCfServiceKey(config abapEnvironmentPullGitRepoOptions, c execRunner) (s
 
 	var abapServiceKey serviceKey
 
-	c.Stderr(log.Entry().Writer())
+	c.Stderr(log.Writer())
```

Comment:
@stippi2 would it make sense to provide a convenience method as part of command.go like `c.StderrToLog()` and similarly `c.StdoutToLog()`. I know it potentially spoils the interface but could reduce the risk of someone falling into using `log.Entry().Writer()`. What do you think?

Reply:
This is what auto-completion currently gives:
![image](https://user-images.githubusercontent.com/59561325/81159313-21e26c80-8f89-11ea-9d11-5fda675da084.png)
... but of course you would have to know to look in log in the first place. I think your proposal makes since, but what do you think about providing a function returning a command instance, which is already fully configured instead (or in addition)?

Reply:
@stippi2 good to see that auto-completion proposes the right things;-). Up to now we do already pre-configure the redirect as part of the construction of command.

The methods would more be for scenarios we i.e. Stdout needs temporary redirection perhaps to capture command output and then afterwards reset it and send it back to the logger.

I'd also be fine to do it in an agile manner, continue as is and optimize based on future demand. Still fairly easy to extend and change that later on.

Reply:
> command.go like c.StderrToLog() and similarly c.StdoutToLog(). I know it potentially spoils the interface

I personally would prefer not keep the interfaces small. In case somebody would like to use another writer etc ... it can simply be provide at c.stderr(MY_WRITER). Less benefit added IMO with additional methods ...